src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -127,6 +126,7 @@ object EasyIngest {
     for {
       foxml <- getFOXML(sdo)
       pid <- executeIngest(configDictionary(sdo.getName), foxml)
+      _ = log.info(s"ingested $pid $foxml")
```

Comment:
It is indeed just the file name that gets logged. I was struggling with the fact that if only some of the files had a bad SHA in the bag, the CompositeException would just report a failure of the files that went wrong and not notify the user other files were ingested. Perhaps it is better to make the composite exception more complete than having the reporting spread this way and run the risk someone someday drops this part. 

Do we want some roll-back for this situation?